src/plugins/tile_map/public/plugin.ts

```diff
@@ -38,7 +38,7 @@ import {
 import { Plugin as ExpressionsPublicPlugin } from '../../expressions/public';
 import { VisualizationsSetup } from '../../visualizations/public';
 // TODO: Determine why visualizations don't populate without this
-import 'angular-sanitize';
+// import 'angular-sanitize';
```

Comment:
@junqiu-lei , since you found no issue on Region map and Coordinate map visualizations, could you confirm that this `import` is no needed and we could remove this completely and also `TODO`?

Reply:
+1. From my local test, I think we can remove the import and TODO.